agreements/ddl.sql

```diff
@@ -1,433 +1,435 @@
-					   
-ALTER TABLE contact_point_lot_prs
-ADD CONSTRAINT contact_point_lot_prs_lot_people_roles_fk FOREIGN KEY (lot_person_role_id)
-    REFERENCES lot_people_roles (lot_person_role_id);
-
-ALTER TABLE contact_point_lot_ors
-ADD CONSTRAINT contact_point_lot_ors_contact_details_fk FOREIGN KEY(contact_detail_id)
-    REFERENCES contact_details (contact_detail_id);
-
-ALTER TABLE contact_point_lot_ors
-ADD CONSTRAINT contact_point_lot_ors_contact_point_reason_fk FOREIGN KEY (contact_point_reason_id)
-    REFERENCES contact_point_reasons (contact_point_reason_id);
-					   
-ALTER TABLE contact_point_lot_ors
-ADD CONSTRAINT contact_point_lot_ors_lot_people_roles_fk FOREIGN KEY (lot_organisation_role_id)
-    REFERENCES lot_organisation_roles (lot_organisation_role_id);
-
-ALTER TABLE contact_point_commercial_agreement_ors
-ADD CONSTRAINT contact_point_commercial_agreement_ors_contact_details_fk FOREIGN KEY(contact_detail_id)
-    REFERENCES contact_details (contact_detail_id);
-
-ALTER TABLE contact_point_commercial_agreement_ors
-ADD CONSTRAINT contact_point_commercial_agreement_ors_contact_point_reason_fk FOREIGN KEY (contact_point_reason_id)
-    REFERENCES contact_point_reasons (contact_point_reason_id);
-					   
-ALTER TABLE contact_point_commercial_agreement_ors
-ADD CONSTRAINT contact_point_commercial_agreement_ors_lot_people_roles_fk FOREIGN KEY (commercial_agreement_organisation_role_id)
-    REFERENCES commercial_agreement_organisation_roles (commercial_agreement_organisation_role_id);
-					  
-ALTER TABLE lot_people_roles 
-ADD CONSTRAINT lot_people_role_people_fk FOREIGN KEY (person_id)
-    REFERENCES people (person_id);						   				   
-
-ALTER TABLE lot_people_roles 
-ADD CONSTRAINT lot_people_roles_people_fk FOREIGN KEY (role_type_id)
-    REFERENCES role_types (role_type_id);	
-					   
-ALTER TABLE people 
-ADD CONSTRAINT people_organisations_fk FOREIGN KEY (organisation_id)
-    REFERENCES organisations (organisation_id);	
-					  
-ALTER TABLE commercial_agreement_benefits
-ADD CONSTRAINT coab_commercial_agreement_fk FOREIGN KEY (commercial_agreement_id)
-    REFERENCES commercial_agreements (commercial_agreement_id);    
-					  
-ALTER TABLE commercial_agreement_updates
-ADD CONSTRAINT coau_commercial_agreement_fk FOREIGN KEY(commercial_agreement_id)
-    REFERENCES commercial_agreements (commercial_agreement_id);    
-
-ALTER TABLE commercial_agreement_documents
-ADD CONSTRAINT coad_commercial_agreement_fk FOREIGN KEY(commercial_agreement_id)
-    REFERENCES commercial_agreements (commercial_agreement_id);    
-
-
-  
+/*
+Title       Database creation script for Agreements database design
+Date        May 26th 2020
+Author      Trevor Cummings
+Description This file is a script to create the tables for the Agreements database. The data model for Agreements will be 
+            deployed in a postgreSQL database.
+            
+Version No Version Description
+---------- -------------------
+V0.0.2      Initial Release
```

Comment:
Tiny point Trev - but maybe should either update the version history or remove it completely (either fine, history is maintained in git anyway)

Reply:
just a note for reference also - the reason this PR suggests that the whole file has changed is because the line endings have changed - I guess because you edited this in Windows, when before it had been edited in Linux (probably via the GitHub UI). We tend to standardise on Linux type line endings. Not an issue at the moment as you are the only person editing these files - just for info in case you were wondering why there was a sea of green changes when only a few lines had actually been altered.